package.json

```diff
@@ -26,21 +25,21 @@
     "lint:sass": "yarn stylelint '{public/sass,packages}/**/*.scss' --cache",
     "test:ci": "mkdir -p reports/junit && JEST_JUNIT_OUTPUT_DIR=reports/junit jest --ci --reporters=default --reporters=jest-junit -w ${TEST_MAX_WORKERS:-100%}",
     "lint:fix": "yarn lint:ts --fix",
-    "packages:build": "lerna run build --ignore '@grafana-plugins/*'",
+    "packages:build": "nx run-many -t build --projects='@grafana/*'",
     "packages:clean": "rimraf ./npm-artifacts && lerna run clean --parallel",
     "packages:prepare": "lerna version --no-push --no-git-tag-version --force-publish --exact",
     "packages:pack": "mkdir -p ./npm-artifacts && lerna exec --no-private -- yarn pack --out \"../../npm-artifacts/%s-%v.tgz\"",
-    "packages:typecheck": "lerna run typecheck",
+    "packages:typecheck": "nx run-many -t typecheck --projects='@grafana/*'",
     "prettier:check": "prettier --check --list-different=false --log-level=warn \"**/*.{ts,tsx,scss,md,mdx,json}\"",
     "prettier:checkDocs": "prettier --check --list-different=false --log-level=warn \"docs/**/*.md\" \"*.md\" \"packages/**/*.{ts,tsx,scss,md,mdx,json}\"",
     "prettier:write": "prettier --list-different \"**/*.{js,ts,tsx,scss,md,mdx,json}\" --write",
-    "start": "yarn themes:generate && yarn dev --watch",
-    "start:noTsCheck": "yarn start --env noTsCheck=1",
-    "start:noLint": "yarn start --env noTsCheck=1 --env noLint=1",
+    "start": "NODE_ENV=dev nx exec -- webpack --config scripts/webpack/webpack.dev.js --watch",
+    "start:noTsCheck": "yarn start -- --env noTsCheck=1",
+    "start:noLint": "yarn start -- --env noTsCheck=1 --env noLint=1",
     "stats": "webpack --mode production --config scripts/webpack/webpack.prod.js --profile --json > compilation-stats.json",
     "storybook": "yarn workspace @grafana/ui storybook --ci",
     "storybook:build": "yarn workspace @grafana/ui storybook:build",
-    "themes:generate": "esbuild --target=es6 ./scripts/cli/generateSassVariableFiles.ts --bundle --platform=node --tsconfig=./scripts/cli/tsconfig.json | node",
+    "themes-generate": "esbuild --target=es6 ./scripts/cli/generateSassVariableFiles.ts --bundle --platform=node --tsconfig=./scripts/cli/tsconfig.json | node",
```

Comment:
It's also a convention in Nx which uses the colon to delimit `nx run {project}:{target}:{configuration}`, so it can interpret the command as a target and configuration. When this happens it can fail to run. A workaround is to wrap the command in quotes but that (IMHO) gets messy. I'm not sure this change is necessary for what I'm proposing in this PR but running the command individually like `yarn nx run grafana:themes:generate` might fail.